hw/top_earlgrey/cdc/cdc_waivers.tcl

```diff
@@ -50,6 +50,131 @@ set_rule_status -rule {W_GLITCH} -status {Waived}          \
     (GlitchOutput=~"*u_sync_1*")} \
   -comment {Waive PADs input goes into synchronizer}
 
+# Driving from PAD RESET to SPI TPM
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"IO*") && (ReceivingFlop=~"*u_spi_device.u_spi_tpm.*")} \
+  -comment {PAD RESET driving to SPI TPM}
+
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"SPI_DEV_CS_L*") && (ReceivingFlop=~"*u_spi_device.u_spi_tpm.*")} \
+  -comment {PAD RESET driving to SPI TPM}
+
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"top_earlgrey.u_pinmux_aon*") && (ReceivingFlop=~"*u_spi_device.u_spi_tpm.*")} \
+  -comment {PAD RESET driving to SPI TPM}
+
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"top_earlgrey.u_rstmgr_aon*") && (ReceivingFlop=~"*u_spi_device.u_spi_tpm.*")} \
+  -comment {PAD RESET driving to SPI TPM}
+
+# Driving from PAD PIN MUX RESET to DMI JTAG
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"top_earlgrey.u_pinmux_aon*") && (ReceivingFlop=~"*u_dmi_jtag.*")} \
+  -comment {PAD RESET driving to dmi jtag}
+
+# dual port memory to SPI
```

Comment:
sorry @jeoongp i can't remember now, but is this one due to RI's issue of not being able to properly recognize the 2p memory? if yes, could you just leave a comment to that? Or add a reference to the issue if it is available?

Reply:
No problem. No, this is apart from not recognizing the 2p memory due to the FPGA style coding. Two paths from different clock domains are muxed and CDC is processed by the following Tx_FIFO. But, the tool seems not to recognize the FIFO after the mux.

Reply:
ah understood. Do you want to add a header comment explaining that? :) it will be very useful to others.